src/util/index.js

```diff
@@ -33,6 +36,47 @@ export function buildContent(options : ContentConfig) : string {
     return content || '';
 }
 
+export function getBuyerCountry(fundingOption : string) : string {
+    switch (fundingOption) {
+    case 'PayUponInvoice': {
+        return 'DE';
+    }
+    case 'boletobancario': {
+        return 'BR';
+    }
+    default: {
+        return 'US';
+    }
+    }
+}
+
+/**
+ * This function defaults the legal text language local based on
+ * payment option and overrides the user preferences
+ * @param {*} fundingOption
+ * @returns
+ */
+export function getLegalLocale(fundingOption : string) : Object  {
+    let locale = {};
+    switch (fundingOption) {
+    case 'PayUponInvoice': {
+        locale = { country: 'DE', lang: 'de' };
+        break;
+    }
+    case 'boletobancario': {
+        locale = { country: 'BR', lang: 'pt' };
+        break;
+    }
+    default: {
+        locale = { country: 'US', lang: 'en' };
+    }
+    }
+    
+    const browserPreference = getLocale() || {};
```

Comment:
getLocales will never return null or undefined, it defaults to `{ country: 'US', lang: 'en' }`